docs/conf/portland/2023/news/writing-day-announcement.rst

```diff
@@ -0,0 +1,98 @@
+
+It's a match made at Write the Docs!
+
+But wait, there's more! This year we have created swag bags, filled with one of a kind items. We'll be gifting them 
+to the project representatives who attend as a thank you.
+
+Get involved with Writing Day!
+------------------------------
+
+Ready to get involved?! `Submit your project <https://forms.gle/NNBzBCwjdB2vF7ZeA>`_ and we'll 
+add it to our `Writing Day project list <https://www.writethedocs.org/conf/portland/2023/writing-day/#project-listing>`__.
+
+Need help convincing your OSS project community or corporate benefactors on the benefits of attending Writing Day? See :ref:`writing-day-case`.
+
+Maximize your experience
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We want you to have a great experience as a project and community. We have found that the projects who
+get the most value out of Writing Day came prepared in the following ways:
+
+* Writing Day label: Creating a specific event label allows you to identify tasks and issues that are good for new contributors.
+* Pre-labeled tasks and issues: Labeling the documentation tasks and issues empowers new contributors 
+  to find tasks that interest them.
+* Task filter: Creating a task filter helps contributors find issues more easily and see which issues have been assigned.
+* Clear onboarding: Ensuring your ReadMe, contribution guidelines, or onboarding instructions are accurate and up to date.
+* Specific focus or goals: Projects that clearly define a focus area or a goal for their Writing Day tasks have 
+  an advantage when it comes to enticing attendees to work on their project.
+* Project overview: Your project overview is a quick pitch that describes your project and what you're hoping to 
+  get out of Writing Day, such as your goals or focus. Need some ideas? Here's a good example from Writing Day 2022, the `Open Web Docs <https://www.writethedocs.org/conf/portland/2022/writing-day/#open-web-docs>`_ project.
+* Writing Day project list: Adding your project to the `Writing Day project list <https://forms.gle/NNBzBCwjdB2vF7ZeA>`_ promotes
+  your project to our attendees before the event. Many attendees have told us that their curiosity about certain projects 
+  incentivised their attendance.
+* Project expert: We recommend having 1-2 project experts of some kind at Writing day. We love our developer advocates, community 
+  managers, and subject matter experts! You're welcome to call for virtual reinforcements from your community as well.
+
+Remember, these are suggestions and not requirements. It is perfectly valid to show up to Writing Day the day of, tell us about 
+your project, and ask for volunteers! It's been done before and it will be done again.
+
+.. note::
+
+   Documentarians have a variety of backgrounds and may need additional info to be successful in onboarding to your project!
+
+.. _writing-day-case:
+
+Make a case for Writing Day
```

Comment:
This could probably migrate into the website as well 💯

Reply:
I agree, I'm hoping to update the Atlantic 2023 Writing Day page with some of the content from this blog post.